spacy_loggers/chain.py

```diff
@@ -0,0 +1,52 @@
+"""
+A utility logger that allows multiple loggers to be daisy-chained.
+"""
+from typing import Dict, Any, Optional, IO
+import sys
+
+from spacy import Language
+from .util import LoggerT
+
+
+def chain_logger_v1(
+    first: LoggerT,
+    second: LoggerT,
+    third: Optional[LoggerT] = None,
+    fourth: Optional[LoggerT] = None,
+    fifth: Optional[LoggerT] = None,
+    sixth: Optional[LoggerT] = None,
+    seventh: Optional[LoggerT] = None,
+    eighth: Optional[LoggerT] = None,
+    ninth: Optional[LoggerT] = None,
+    tenth: Optional[LoggerT] = None,
```

Comment:
Lists of things is indeed tricky.

I think something like `logger1`, `logger2`, ... might be simpler? (No idea whether you want `logger0`.)

I don't any particular reason to have a difference in which ones are optional or not?

Reply:
Sure, `logger1`, `logger2`, ... sounds fine to me.

My thinking with the optional params was that it'd only make sense to use the chain logger if one had at least two individual loggers.